monitoring/config.go

```diff
@@ -0,0 +1,9 @@
+package monitoring
+
+// PrometheusConfig is the set of configuration data that specifies
```

Comment:
Alternatively, this can live in `lncfg` and have two versions: one with w/ the build tag, and one w/o it. This way, if the user doesn't have this tag enabled, then it won't show up in `lnd -h`.

Reply:
IIUC, fixed.